### ISSUE TITLE
Respect "Now as Default" setting for Date/Time columns

### ISSUE THREAD

**Issue by lhupitr** [stefan-niedermann/nextcloud-tables#19]
Please use **[GitHub reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** 👍 to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!

**Is your feature request related to a problem? Please describe.**

When a date/time column is set to "Now as Default" on the web app the Android app still requires a date and time to be entered manually.

**Describe the solution you'd like**

Make the Android app respect the "Now as Default" setting so that multiple screen taps are not necessary to enter the current date/time,

**Comment by stefan-niedermann** [comment]
We actually have implemented this logic for `datetime` and the subtypes `datetime`, `date` and `time`. However, `date` reacts to `"today"` while the others react to `"now"` as default value. I therefore need more information about the table you set up, especially the column where prefilling does not work. What exact type is it? Maybe a screenshot of the Web UI configuration page might help.

**Comment by lhupitr** [comment]
Seems to be working now if I have separate date & time as outlined here:
https://github.com/stefan-niedermann/nextcloud-tables/issues/38

**Comment by stefan-niedermann** [comment]
So what about `datetime`? I think this issue was about `datetime` tyoe?

**Comment by lhupitr** [comment]
Yes datetime enters everything correctly by default it just disappears on synchronisation as outlined:
https://github.com/stefan-niedermann/nextcloud-tables/issues/38

**Comment by stefan-niedermann** [comment]
Should be fixed in `2.0.2`. Please reopen in case the issue still occurs.
